src/agent/Cargo.toml

```diff
@@ -60,6 +60,7 @@ vsock-exporter = { path = "vsock-exporter" }
 # Configuration
 serde = { version = "1.0.129", features = ["derive"] }
 toml = "0.5.8"
+clap = { version = "3.0.0-rc.4", features = ["derive"] }
```

Comment:
You could bump this to `3.0.0-rc.7` now.

Reply:
Thanks, I've done that, so will re-run the tests :)

Reply:
clap `3.0.1` has released now, so I've updated to that and will run the tests again.